mysqld_exporter.go

```diff
@@ -656,6 +674,11 @@ type Exporter struct {
 	totalScrapes    prometheus.Counter
 }
 
+// Last run tracking variable for scrapeTableSchema()
+var (
```

Comment:
Terminate comment with `.`
A single var shouldn't have a `()` block for consistency.